pos_customer_required/static/src/js/Screens/PartnerListScreen/PartnerListScreen.esm.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+/* Copyright CoopITEasy - Simon Hick <sim@coopiteasy.be>
+   License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl) */
+
+import PartnerListScreen from "point_of_sale.PartnerListScreen";
+import Registries from "point_of_sale.Registries";
+
+const PosCustomerRequiredPartnerListScreen = (OriginalPartnerListScreen) =>
+    class extends OriginalPartnerListScreen {
+        clickPartner(partner) {
+            if( this.env.pos.config.require_customer === "order" ) {
```

Comment:
spacing: should be `if (condition) {` instead of `if( condition ) }`. `if` is not a function, and there should never be spaces after an opening parenthesis or before a closing one.